content/html/concepts/semantic-html/terms/rb/rb.md

```diff
@@ -0,0 +1,49 @@
+---
+Title: '<rb>'
+Description: 'Serving as the base component in ruby annotations for East Asian text, this element's use has declined and is now listed as deprecated.'
+Subjects:
+  - 'Web Development'
+  - 'Web Design'
+Tags:
+  - 'Semantic'
+  - 'Tags'
+CatalogContent:
+  - 'learn-html'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The **`<rb>`** element designates the base text for ruby annotations in East Asian languages. It must be within `<ruby>` tags and is often paired with `<rt>` for pronunciation guides and `<rp>` for fallback rendering.
```

Comment:
[Proofreading] Always hyperlink elements that correlate to the one you're writing about. 
```suggestion
The **`<rb>`** element designates the base text for ruby annotations in East Asian languages. It must be within `<ruby>` tags and is often paired with [`<rt>`](https://www.codecademy.com/resources/docs/html/semantic-html/rt) for pronunciation guides and `<rp>` for fallback rendering.
```